Assets/Scripts/Battle/UIController.cs

```diff
@@ -1041,6 +1043,48 @@ public enum UIState {
             PlaySound(AudioClipRegistry.GetSound("menuconfirm"));
     }
 
+    public static void BlockButton(string btn)
+    {
```

Comment:
Please follow the same coding style as the rest of the engine (curly braces on the same line as the block using it, etc)

Reply:
Hello. I didn't know about the braces, will fix it soon